src/trip/components/tripsTable.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from 'react'
+import { getCredentials, register, unregister } from '../../login/store'
+
+import { Table, Button } from 'semantic-ui-react'
+import dateFormat from '../../utils/dateFormat'
+
+const TripsTable = ({ trips, isBack }) => {
+    const [title] = useState(isBack ? 'Trajet retour' : 'Trajet Aller')
+    const [credentials, setCredentials] = useState()
+    const [connectedUser, setConnectedUser] = useState()
+
+    useEffect(() => {
+        console.log('did mount: ', credentials)
+        register(setCredentials)
+        setCredentials(getCredentials())
+    }, [])
+     
+    useEffect(() => {
+        console.log('credentials did update: ', credentials)
+        if (credentials) setConnectedUser(credentials.login ? credentials.login : undefined)
+    }, [credentials])
+
+    return (
+        <Table celled striped>
+            <Table.Header>
+                <Table.Row>
+                    <Table.HeaderCell colSpan='5'>
+                        <>{title}</>
+                        <Button floated='right' primary>Ajouter</Button>
```

Comment:
Maybe "Nouveau" would make more sense from the user than "Ajouter" :)

Reply:
I agree with that.